### DISCUSSION TITLE
Connection status in a combobox

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#3864] [General] [answered]
Hey team!

Do we have a preferred pattern for showing connection status in a combobox?

**Scenario**: A user connected their GitHub account while going through the dbt connection process. Now they want to set up Git Sync using the same GitHub credentials. It would be nice to show them in the combobox that this connection has already been made, and they won’t have to reenter their credentials:

![Screenshot 2024-04-16 at 9 43 59 AM](https://github.com/twilio-labs/paste/assets/87141551/702bb005-704c-4bc1-a1d5-cb0bff05aba2)

Thank you!!

**Comment by serifluous** [comment, marked as answer]
Hey @ceyerkes ! That's definitely the preferred style to add extra information to a combobox option. In this case, though, if you don't feel like it's super easily scannable, you could consider adding an icon in front of the word "Connected". Or adding some sort of indicator (icon, badge, etc) to the right side of the option.

The layout of the combobox option is pretty flexible. You can see an example here: https://paste.twilio.design/components/combobox#combobox-using-option-template

**Reply by ceyerkes** [reply]
sounds great, thanks @serifluous!
